extension/src/agent/v1/api-handler.ts

```diff
@@ -415,16 +414,24 @@ ${this.customInstructions.trim()}
 			estimateTokenCount(history[history.length - 1])
 
 		let contextWindow = this.api.getModel().info.contextWindow
+		// Check if the context window needs to be compressed, if so compress it otherwise do nothing
+		const maxPostTruncationTokens = contextWindow - 13_314 + this.api.getModel().info.maxTokens
+
+		if (totalTokens > maxPostTruncationTokens) {
```

Comment:
this should never be true as we call this method only when we reached error from anthropic regarding the context window.